surface_apps/driver.py

```diff
@@ -45,11 +45,11 @@ def __init__(self, parameters: BaseData | InputFile):
     def out_group(self) -> UIJsonGroup | None:
         """Output container group."""
 
-        if self._out_group is None and self.params.output.out_group is not None:
+        if self._out_group is None:
             with fetch_active_workspace(self.workspace, mode="r+") as workspace:
                 self._out_group = UIJsonGroup.create(
                     workspace=workspace,
-                    name=self.params.output.out_group,
+                    name=self.params.title,
```

Comment:
why using the title of the uijson rather the name of the object to create?

Just for my understanding.

Reply:
It sets the title of the group, which is different from the output object
![image](https://github.com/user-attachments/assets/552089ea-3bf4-4277-aa52-99fd1206229e)